### PR TITLE
Add AdoptOpenJDK 8u222 with Hotspot and OpenJ9

### PR DESCRIPTION
**Pull request by joschi**

### CODE REVIEW

File: src/main/scala/io/sdkman/changelogs/JavaMigrations.scala [hunk 1]
```diff
@@ -451,4 +451,26 @@ class JavaMigrations {
       .insert()
     Seq(Linux64, MacOSX, Windows).foreach(removeVersion("java", "14.ea.5-open", _))
   }
+
+  @ChangeSet(order = "169", id = "169-add_adoptopenjdk-hs_8_0_222", author = "jschalanda")
+  def migrate169(implicit db: MongoDatabase) = {
+    List(
+      Version("java", "8.0.222.hs-adpt", "https://github.com/AdoptOpenJDK/openjdk8-binaries/releases/download/jdk8u222-b10/OpenJDK8U-jdk_x64_linux_hotspot_8u222b10.tar.gz", Linux64),
```
Comment (marc0der) [inline src/main/scala/io/sdkman/changelogs/JavaMigrations.scala]: We require a vendor to be added the end of every Java migration now. Could I ask you to add that to all your new versions please?

File: src/main/scala/io/sdkman/changelogs/JavaMigrations.scala [hunk 2]
```diff
@@ -484,4 +484,26 @@ class JavaMigrations {
...
+    List(
+      Version("java", "8.0.222.hs-adpt", "https://github.com/AdoptOpenJDK/openjdk8-binaries/releases/download/jdk8u222-b10/OpenJDK8U-jdk_x64_linux_hotspot_8u222b10.tar.gz", Linux64, Some(AdoptOpenJDK)),
+      Version("java", "8.0.222.hs-adpt", "https://github.com/AdoptOpenJDK/openjdk8-binaries/releases/download/jdk8u222-b10/OpenJDK8U-jdk_x64_mac_hotspot_8u222b10.tar.gz", MacOSX, Some(AdoptOpenJDK)),
+      Version("java", "8.0.222.hs-adpt", "https://github.com/AdoptOpenJDK/openjdk8-binaries/releases/download/jdk8u222-b10/OpenJDK8U-jdk_x64_windows_hotspot_8u222b10.zip", Windows, Some(AdoptOpenJDK)))
+      .validate()
+      .insert()
+    Seq(Linux64, MacOSX, Windows).foreach(removeVersion("java", "8.0.212.hs-adpt", _))
+  }
+
+  @ChangeSet(order = "173", id = "173-add_adoptopenjdk-hs_8_0_222", author = "jschalanda")
```
Comment (tkp1n) [inline src/main/scala/io/sdkman/changelogs/JavaMigrations.scala]: Possible copy/paste error in `id` of ChangeSet... should be 173-add_adoptopenjdk-**j9**_8_0_222

**Comment by tkp1n** [comment]
Would you please also add AdoptOpenJDK 11.0.4 while you're on it? I don't want to open a separate PR with then conflicting ChangeSet ids...

### hs
https://github.com/AdoptOpenJDK/openjdk11-binaries/releases/download/jdk-11.0.4+11/OpenJDK11U-jdk_x64_linux_hotspot_11.0.4_11.tar.gz
https://github.com/AdoptOpenJDK/openjdk11-binaries/releases/download/jdk-11.0.4+11/OpenJDK11U-jdk_x64_mac_hotspot_11.0.4_11.tar.gz
https://github.com/AdoptOpenJDK/openjdk11-binaries/releases/download/jdk-11.0.4+11/OpenJDK11U-jdk_x64_windows_hotspot_11.0.4_11.zip

### j9
https://github.com/AdoptOpenJDK/openjdk11-binaries/releases/download/jdk-11.0.4+11_openj9-0.15.1/OpenJDK11U-jdk_x64_linux_openj9_11.0.4_11_openj9-0.15.1.tar.gz
https://github.com/AdoptOpenJDK/openjdk11-binaries/releases/download/jdk-11.0.4+11_openj9-0.15.1/OpenJDK11U-jdk_x64_mac_openj9_11.0.4_11_openj9-0.15.1.tar.gz
https://github.com/AdoptOpenJDK/openjdk11-binaries/releases/download/jdk-11.0.4+11_openj9-0.15.1/OpenJDK11U-jdk_x64_windows_openj9_11.0.4_11_openj9-0.15.1.zip

File: src/main/scala/io/sdkman/changelogs/JavaMigrations.scala (hunk 2 above)
Comment (joschi) [reply src/main/scala/io/sdkman/changelogs/JavaMigrations.scala]: Good catch, thanks!

**Comment by joschi** [comment]
@tkp1n I've also added AdoptOpenJDK 11.0.4.

**Comment by marc0der** [comment]
We now just need to update the changelog numbers to fit into the sequence. Sorry!

**Comment by joschi** [comment]
@marc0der I've resolved the merge conflict.

**Comment by marc0der** [comment]
Thanks!

**Pull request merged**